src/scss/layout/_footer.scss

```diff
@@ -0,0 +1,48 @@
+.footer {
+  background: #000;
+  padding: 59px 91px 64px 30px;
+
+&__logo {
```

Comment:
Please add indentation to the classes inside the `.footer` class

Reply:
The indentation was already added